legal_db/static/styles.scss

```diff
@@ -207,7 +207,7 @@ a, a:hover {
 }
 
 footer a:hover {
-  color: $color-dark-turquoise;
+  color: $color-forest-green;
```

Comment:
This must keep the turquoise color.

Reply:
Ok i will look into it

Reply:
> This must keep the turquoise color.

do I  only need to change it back to dark-turquoise on the  hover on the footer